core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -303,6 +303,8 @@ oppia.directive('conversationSkin', [
           $scope.hasFullyLoaded = false;
           $scope.recommendedExplorationSummaries = null;
           $scope.answerIsCorrect = false;
+          $scope.conceptCardShown = false;
```

Comment:
Perhaps conceptCardIsBeingShown to make it clear it's not past tense.

Reply:
Done.